kloppy/domain/models/event.py

```diff
@@ -130,6 +130,17 @@ class CardType(Enum):
     RED = "RED"
 
 
+class FormationType(Enum):
+    """
+    FormationType
+
+    Attributes:
+        FORMATION_TYPE (FormationType): formation the team changes to
+    """
+
+    FORMATION_TYPE = None
```

Comment:
This should include the list of formations. This list: https://github.com/PySport/kloppy/pull/125/files#diff-6b63d1c70bf1c1cdc45872bdf3bebb33bb5b9696761ffff0f16e66d2a5538fe5R194